services/motion/builtin/builtin.go

```diff
@@ -214,6 +214,16 @@ func (ms *builtIn) Move(
 	}
 	goalPose, _ := tf.(*referenceframe.PoseInFrame)
 
+	// Check that we have some planDeviationMM value for our requested motion
```

Comment:
Why would we be adding this here?

Reply:
checking that we are not planning from the goal happens in the motionplan package. since it happens there we need some value for planDevMM for all possible move calls, this means having a planDevMM value for bases, arms, grippers, etc...